packages/react-search-ui/src/containers/ResultsPerPage.js

```diff
@@ -8,6 +8,7 @@ export class ResultsPerPageContainer extends Component {
     // Props
     className: PropTypes.string,
     view: PropTypes.func,
+    resultsPerPageOptions: PropTypes.arrayOf(PropTypes.number),
```

Comment:
Would it be better to call this `option`? Having a property `resultsPerPageOptions` on a `ResultsPerPage` component sounds redundant.

Reply:
Yes, Make sense.